@xen-orchestra/lite/src/components/Tree.tsx

```diff
@@ -73,21 +73,26 @@ interface Computed {}
 // Inspired by https://mui.com/components/tree-view/#contentcomponent-prop.
 const CustomContent = React.forwardRef(function CustomContent(props: CustomContentProps, ref) {
   const { classes, className, label, expansionIcon, nodeId, to } = props
-  const { handleExpansion, handleSelection, selected } = useTreeItem(nodeId)
+  const { focused, handleExpansion, handleSelection, selected } = useTreeItem(nodeId)
   const history = useHistory()
 
+  React.useEffect(() => {
+    if (selected) {
+      to !== undefined && history.push(to)
```

Comment:
What should be the url if nothing is selected   ?

Reply:
When mounting the `Tree` component with no selected VM, it will be `/#/infrastructure`